src/Confluent.Kafka/SysLogLevel.cs

```diff
@@ -0,0 +1,41 @@
+namespace Confluent.Kafka
+{
+    /// <summary>
+    ///  Represents enumeration with levels comming from syslog(3)
+    /// </summary>
+    public enum SysLogLevel
```

Comment:
nit: Since "syslog" is a well known entity and is always spelled as "syslog" (or sometimes "Syslog") but never "SysLog", I think this should be changed to "SyslogLevel".